pkg/util/overridemanager/overridemanager_test.go

```diff
@@ -14,6 +19,195 @@ import (
 	"github.com/karmada-io/karmada/test/helper"
 )
 
+func Test_overrideManagerImpl_ApplyOverridePolicies(t *testing.T) {
+	deployment := helper.NewDeployment(metav1.NamespaceDefault, "test1")
+	deployment.Labels = map[string]string{
+		"foo": "foo",
+	}
+	deploymentObj, _ := utilhelper.ToUnstructured(deployment)
+
+	clusterRole := helper.NewClusterRole("test2", nil)
+	clusterRole.Labels = map[string]string{
+		"bar": "bar",
+	}
+	clusterRoleObj, _ := utilhelper.ToUnstructured(clusterRole)
+	type fields struct {
+		Client        client.Client
+		EventRecorder record.EventRecorder
+	}
+	type args struct {
+		rawObj      *unstructured.Unstructured
+		clusterName string
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		want    *AppliedOverrides
+		want1   *AppliedOverrides
```

Comment:
I suggest naming them `wantCOP` and `wantOP` respectively.

Reply:
ok